maven-jaxb-pluging-test/importing-xsd/src/main/resources/xsd/importing.xsd

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<schema
+  xmlns="http://www.w3.org/2001/XMLSchema"
+  targetNamespace="http://my.uri/importingXsd"
+  xmlns:import="http://my.uri/commonDefinitions"
+  xmlns:jxb="http://java.sun.com/xml/ns/jaxb"
+  jxb:version="1.0"
+  elementFormDefault="qualified">
+ <!-- direct relative reference to be resolved - ->
+  <import
+    namespace="http://my.uri/commonDefinitions"
+    schemaLocation="../../../../../common-xsd/src/main/resources/xsd/common.xsd" />
+--> 

```

Comment:
activating the upper `import` and deactivating the lower makes the testcase build successfully.